spring-boot-admin-server/src/test/java/de/codecentric/boot/admin/discovery/eureka/EurekaDiscoveryListenerTest.java

```diff
@@ -0,0 +1,85 @@
+package de.codecentric.boot.admin.discovery.eureka;
+
+import de.codecentric.boot.admin.config.EnableAdminServer;
+import de.codecentric.boot.admin.event.ClientApplicationRegisteredEvent;
+import de.codecentric.boot.admin.model.Application;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.boot.context.embedded.EmbeddedWebApplicationContext;
+import org.springframework.cloud.netflix.eureka.EnableEurekaClient;
+import org.springframework.cloud.netflix.eureka.server.EnableEurekaServer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.event.EventListener;
+import org.springframework.util.concurrent.SettableListenableFuture;
+
+import java.util.concurrent.ExecutionException;
+
+/**
+ * Created by alexf on 11-Feb-16.
+ */
+
+public class EurekaDiscoveryListenerTest {
+
+    private EmbeddedWebApplicationContext eurekaServer;
+
+    @Before
+    public void setup() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [You may have misspelled a JUnit framework method (setUp or tearDown)](https://www.codacy.com/app/joshiste/spring-boot-admin/file/2229709767/issues/source?bid=3175464&fileBranchId=3175465#l30)